### PR TITLE
Add Unit test in `controllers/timechaos`

### PR DESCRIPTION
**Pull request by oraluben**
UCP #265

### What problem does this PR solve? 
#265

### Check List 

Tests 

 - Unit test

Code changes

 - Has Go code change

Side effects

Related changes

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

**Comment by oraluben** [comment]
@fewdan PLTA if the overall structure is ok, mainly about how `MockChaosDaemonClient` works.
https://github.com/pingcap/chaos-mesh/pull/314/files#diff-0fb24aea9516170aab0542b352b676c7L235-R274

**Comment by oraluben** [comment]
@fewdan PLTA if the overall structure is ok, mainly about how `MockChaosDaemonClient` works.
https://github.com/pingcap/chaos-mesh/pull/314/files#diff-0fb24aea9516170aab0542b352b676c7L235-R274

**Comment by fewdan** [comment]
@oraluben Thank you for your contribution. I think your structure is fine.

**Comment by oraluben** [comment]
@fewdan PTAL

File: controllers/timechaos/timechaos_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/pingcap/chaos-mesh/api/v1alpha1"
+	. "github.com/pingcap/chaos-mesh/controllers/timechaos"
+	"github.com/pingcap/chaos-mesh/pkg/mock"
+
+	. "github.com/pingcap/chaos-mesh/controllers/test"
+)
```
Comment (fewdan) [inline controllers/timechaos/timechaos_suite_test.go]: please group go imports

**Comment by zhouqiang-cl** [comment]
@lucklove PTAL too

**Pull request merged**